src/python/pants/notes/2.11.x.md

```diff
@@ -1,5 +1,63 @@
 # 2.11.x Release Series
 
+## 2.11.0.dev1 (Feb 22, 2022)
+
+### New Features
+
+* Support for `file`/`files` in JVM tests ([#14537](https://github.com/pantsbuild/pants/pull/14537))
+
+* Support complete_platforms for `python_awslambda`. ([#14532](https://github.com/pantsbuild/pants/pull/14532))
+
+* Support complete_platforms for Python GCFs. ([#14530](https://github.com/pantsbuild/pants/pull/14530))
+
+* Properly support filesystem specs in the `paths` goal. ([#14518](https://github.com/pantsbuild/pants/pull/14518))
+
+* Plumb Pex's `--complete-platform` for pex_binary. ([#14526](https://github.com/pantsbuild/pants/pull/14526))
+
+* Tests scala local plugins, and makes it work properly on `scala_sources` targets ([#14512](https://github.com/pantsbuild/pants/pull/14512))
+
+* Allow use of `parametrize` with the `jdk=` field of JVM targets ([#14510](https://github.com/pantsbuild/pants/pull/14510))
+
+* Adds the plugin names field to scala sources ([#14508](https://github.com/pantsbuild/pants/pull/14508))
+
+* Per-resolve scala plugins ([#14497](https://github.com/pantsbuild/pants/pull/14497))
+
+* jvm: teach dependency inference about multiple resolves ([#14491](https://github.com/pantsbuild/pants/pull/14491))
+
+* Add disclaimer about `[python].enable_resolves` and lockfile generation ([#14475](https://github.com/pantsbuild/pants/pull/14475))
+
+* scala: add implicit dependency on scala-library ([#14450](https://github.com/pantsbuild/pants/pull/14450))
```

Comment:
This seems like a bug fix, but maybe eye of the beholder. Having to both say `scala_junit_test` _and_ add a dependency on the scala library seems like a bug to me. Perhaps this was just calling our shots and shipping smaller to ship before fixing later.

Reply:
Yeah, I can see both sides. I think I'll leave it though.